gen/aws/templates/cloudformation.json

```diff
@@ -317,7 +317,7 @@
         "SourceDestCheck" : "false",
         "KeyName" : { "Ref" : "KeyName" },
         "ImageId" : { "Fn::FindInMap" : [ "NATAmi", { "Ref" : "AWS::Region" }, "default" ] },
-        "InstanceType" : "m3.medium",
+        "InstanceType" : "m4.large",
```

Comment:
can you change this back to `m3.medium` I think this is what is breaking creation right now.